.github/workflows/check-format.yml

```diff
@@ -16,6 +16,8 @@ jobs:
     runs-on: ubuntu-latest
     container:
       image: ghcr.io/llnl/sundials_spack_cache:llvm-17.0.4-h4lflucc3v2vage45opbo2didtcuigsn.spack
+    permissions:
+      contents: write
```

Comment:
Does this need to be added to the `apply_format` job as well?

Reply:
It only needs to be under `apply_format`. I pasted it into the wrong job.